app/models/review.rb

```diff
@@ -1,5 +1,8 @@
 class Review < ApplicationRecord
-  belongs_to :user
+  validates :rating, presence: true, numericality: {greater_than_or_equal_to: 0, less_than_or_equal_to: 5}
+  validates :body, presence: true
+
   belongs_to :fruit
-  validates :rating, presence: true
+  has_many :users, through: :reviews
+  has_many :countries, through: :fruits_of_countries
 end
```

Comment:
Your associations are a bit off here. 

A review should belong to both a user and a fruit
if a review `has_many` `:countries, through: :fruits_of_countries` then it also must `has_many :fruits of countries`

Reply:
removed the whole link to the countries b/c we're probably not getting there anyway. Then added the belongs_to :user.